Application/EdFi.Ods.AdminApi/Docker/dev/run.sh

```diff
@@ -5,7 +5,7 @@
 # See the LICENSE and NOTICES files in the project root for more information.
 
 set -e
-set -x
+set +x
```

Comment:
We disabled echoing the whole command in the postgresql container's run script, so I changed it elsewhere for consistency.